src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/Button.xaml

```diff
@@ -93,6 +93,7 @@
                                 <Condition Property="IsMouseOver" Value="True" />
                                 <Condition Property="IsPressed" Value="True" />
                             </MultiTrigger.Conditions>
+                            <Setter Property="BorderThickness" Value="0" />
```

Comment:
@anjali-wpf This change does not actually fix the pressed state issue. The Button is no longer distinguishable from its background when applied on top standard window, given that its border no longer exists.